tests/common/libvm.sh

```diff
@@ -34,6 +34,25 @@ vm_setup() {
 
   export SSH="ssh $sshopts $VM"
   export SCP="scp $sshopts"
+
+  cat >${topsrcdir}/ansible-inventory.yml <<EOF
```

Comment:
Hmm, I'm not sure if this is working properly judging from the test logs:

```

PLAY [vmcheck2] ****************************************************************
skipping: no hosts matched
```

OTOH, you can probably just drop this and use

```
ansible-playbook -i ${VM},  --ssh-common-args "${sshopts}" ${playbook}
```

?